release_dashboard/lib/widgets/common/url_launch_inbrowser.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:url_launcher/link.dart';
+
+/// Clicks on the text displayed will open the URL on a web browser.
```

Comment:
Adding more detail through square brackets would be helpful. `Constructs a [Button] that displays [textToDisplay].`